paywall/src/data-iframe/blockchainHandler/account.js

```diff
@@ -39,6 +39,7 @@ export async function pollForAccountChange(
         newAccount ? await web3Service.getAddressBalance(newAccount) : 0
       )
       onAccountChange(account, accountBalance)
-    }
+    } /*changeListener */,
+    5000 /* delay */
```

Comment:
How would you feel about pulling the `5000` to a variable or constant? Also why 5 seconds?

Reply:
5 seconds was arbitrary. We have a constant for this, I just updated to use it